inc/inventoryruleimport.class.php

```diff
@@ -882,7 +882,8 @@ function findWithGlobalCriteria($input) {
                               && !is_null($value)
                               && is_numeric($value)
                               && $value > 0) {
-                           $this->criterias_results['found_port'] = $value;
+                              $this->criterias_results['found_port'] = $value;
```

Comment:
The indentation was good, 3 spaces from the `if (` ;)

Reply:
indeed, but next line was missing (9.5 branch is currently broken)